examples/simple-chatbot/client/javascript/src/app.js

```diff
@@ -42,6 +57,7 @@ class ChatbotClient {
     this.statusSpan = document.getElementById('connection-status');
     this.debugLog = document.getElementById('debug-log');
     this.botVideoContainer = document.getElementById('bot-video-container');
+    this.deviceSelector = document.getElementById('device-selector');
```

Comment:
i've also added a device selector to the sample because i was tired of not having it 😂